src/cli.rs

```diff
@@ -108,6 +113,11 @@ enum Commands {
         /// A payer-provided note which will be seen by the recipient.
         #[arg(required = false)]
         payer_note: Option<String>,
+
+        /// The amount of time in seconds that the user would like to wait for an invoice to
+        /// arrive. If this isn't set, we'll use the default value.
+        #[arg(long, global = false, required = false, default_value = "15")]
```

Comment:
Let's use the DEFAULT_RESPONSE_INVOICE_TIMEOUT here and above

Reply:
Good catch. Updated in squashed commit

Reply:
Thanks! Looks like it just needs a cargo fmt